### DISCUSSION TITLE
How can I read "address" from "PlatformAdvertisement" in the iOS module?

### DISCUSSION THREAD

**Discussion by surame8288** [JuulLabs/kable#715] [Q&A] [answered]
I'm working on the "Compose Multiplatform Project."How can I read "address" from PlatformAdvertisement in the iOS module?

**Comment by twyatt** [comment, marked as answer]
Unfortunately, Core Bluetooth (iOS) does not expose the peripheral's MAC address.

> The CoreBluetooth APIs in iOS do not give you any direct access to a peripheral's real MAC address; instead, each new peripheral seen is assigned a 128-bit UUID and used to reference that peripheral

— [HOW-TO: Get a BLE peripheral MAC address with iOS and CoreBluetooth](https://community.silabs.com/s/article/x-how-to-get-a-ble-peripheral-mac-address-with-ios-and-corebluetooth?language=en_US)

**Reply by surame8288** [reply]
Thank you for the confirmation.
